lib/libutee/include/tee_api_defines.h

```diff
@@ -232,12 +232,14 @@
 #define TEE_ALG_RSASSA_PKCS1_V1_5_SHA384        0x70005830
 #define TEE_ALG_RSASSA_PKCS1_V1_5_SHA512        0x70006830
 #define TEE_ALG_RSASSA_PKCS1_V1_5_MD5SHA1       0x7000F830
+#define TEE_ALG_RSASSA_PKCS1_PSS_MGF1_MD5       0x70111930
 #define TEE_ALG_RSASSA_PKCS1_PSS_MGF1_SHA1      0x70212930
 #define TEE_ALG_RSASSA_PKCS1_PSS_MGF1_SHA224    0x70313930
 #define TEE_ALG_RSASSA_PKCS1_PSS_MGF1_SHA256    0x70414930
 #define TEE_ALG_RSASSA_PKCS1_PSS_MGF1_SHA384    0x70515930
 #define TEE_ALG_RSASSA_PKCS1_PSS_MGF1_SHA512    0x70616930
 #define TEE_ALG_RSAES_PKCS1_V1_5                0x60000130
+#define TEE_ALG_RSAES_PKCS1_OAEP_MGF1_MD5       0x60110230
```

Comment:
Both defines should go in `tee_api_defines_extensions.h` because they are non-standard (not in the GlobalPlatform spec).

Reply:
Applied, Thank you for the review.